docs/packaging-an-application/components-and-containers.md

```diff
@@ -193,6 +193,31 @@ Optional properties:
       is_excluded_from_backup: true
 ```
 
+We can use "volumes from" to attach several mounts from a colocated container.
+
+```yml:
+  - source: public
+    ...
+    name: datastore
+    publish_events:
+    - name: Datastore started
+      trigger: port-listen
+      data: '6379'
+      subscriptions:
+      - component: DBs
+        container: alpine
+        action: start
+  - source: public
+    image_name: alpine
+    version: 3
+    ephemeral: true
+    cmd: '["migrate_data.sh"]'
+    volumes_from: ["datastore"]
+```
+
+The container using "volumes from" must start after any containers it mounts from.  Volumes from takes an array of
```

Comment:
`The container using  "volumes_from" must start after any containers it mounts from.  Property "volumes_from" takes...`